version.gradle.kts

```diff
@@ -40,12 +40,12 @@
 /**
  * Version of this library.
  */
-val time = "2.0.0-SNAPSHOT.18"
+val time = "2.0.0-SNAPSHOT.20"
```

Comment:
Can we use `2.0.0-SNAPSHOT.18`  version for `time` as well?

Reply:
Unfortunately, no. It's gone ahead already. I'm going to level it up soon.